src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery.getRandomBall();
```

Comment:
use System.out.println() to print received balls

Reply:
you have not fixed this, the method in Lottery should not be static, 
create a variable lottery outside of for loop
```suggestion
           System.out.println(lottery.getRandomBall());
```